### PR TITLE
Refactor to classes

### PR DESCRIPTION
**Pull request by emiliazm**
Hello!

# These are the changes that we implemented:
- Use JavaScript classes instead of objects and arrays.
- Create class methods to add and remove books.
- Add CSS styles to the application to make it match the wireframe.

### CODE REVIEW

File: classes/ui.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    card.querySelector('button').addEventListener('click', (event) => UI.removeBook(event, book));
+    this.#bookListElement.appendChild(card);
+  }
+
+  static removeBook(event, book) {
+    const buttonElement = event.target;
+    const cardElement = buttonElement.parentElement;
+    cardElement.remove();
+    BookCollection.remove(book.id);
+  }
```
Comment (brytebee) [inline classes/ui.js:20]: - [ ] **Required:** You have done an amazing job so far. 👏 
However, your local storage is not working properly. When an item is removed, your entire store is broken. Kindly fix this behaviour.

remove empties local storage2

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @emiliazm & @rbanos-mv👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- Refactored app with classes.✔️
- It has class methods to add and remove books.✔️
- No console errors.✔️
- Linters are properly set up and green.✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by emiliazm** [comment]
Hello @brytebee !
Thank you for your comments.

# Changes applied:
-Fixed the storage problem. (The key for saving local storage was different from the key used for reading local storage.)

**Review by brytebee — APPROVED** [review]
Hi, @emiliazm & @rbanos-mv 👋, 

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**